src/components/tuTable/tuTable.vue

```diff
@@ -61,10 +61,16 @@
 							minWidth: header.minWidth,
 							maxWidth: header.maxWidth
 						}"
+                        draggable="true"
```

Comment:
can we make this draggable dynamic ? Make it true only if a prop is true ?

something like 
```js
    :draggable="isDraggable"
    
    ..
    ..
    props:  {
        isDraggable: Boolean
    }
```

Reply:
Okay